moi/__init__.py

```diff
@@ -16,6 +16,24 @@
 from moi.context import Context  # noqa
 
 
+def _support_directory():
+    """Get the path of the support_files directory"""
+    from os.path import join, dirname, abspath
```

Comment:
Since this is in init, the motivation is to minimize overhead as those
imports are only necessary if those functions are called but no issue to
move if you prefer
On May 29, 2015 8:24 AM, "adamrp" <notifications@github.com> wrote:

> In moi/__init__.py
> <https://github.com/biocore/mustached-octo-ironman/pull/28#discussion_r31334619>
> :
>
> > @@ -16,6 +16,24 @@
> >  from moi.context import Context  # noqa
> >
> >
> > +def _support_directory():
> > +    """Get the path of the support_files directory"""
> > +    from os.path import join, dirname, abspath
>
> PEP-8 suggests that imports should always go at the top of the file
> <https://www.python.org/dev/peps/pep-0008/#imports>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/mustached-octo-ironman/pull/28/files#r31334619>
> .
>